domains/asteral.json

```
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "Asteral1",
+           "email": "",
+           "discord": "870366927653056582",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.F2v6VwlZy8PN3eAm0sdM9Kqvh7Cah2AqvxrUMBqNmUI0p-bZbD4Ovfjm2Ti_5SOJB2CF_Ff-2Fb1oaJfef7OeUL9YIdu2O2jFqUR81znaNuPoT9VpXEIAeIQwBvQylSRqnhZK92QALGNUOIo8_9scBEZ-4K1Sfgn6ypISO-O5en13fcYW5gjX_CdAy1CjzHU7PZ0EuF-Ym0G5FXj6gGY3AbysoSDAef38r7uzFbUKEbcRd6QUjMoQkGp9ST9NqP_-Mg0XFEPC9Bbyc6lvGwe2DbIoNzUqlo3b6nViKSiS0UIXe_cT0aImPRqvR4N5wSvwR3XK1R2mY0EewgsTjq7vg.7AhkpT73ke6dCcN9XNwSMw.E_L4WPiryeVCW_0jWiWMA6QjWu5gIIA4Ipysar9kOANIogUjr8Gwfxrj-0rW9h7gfeSbFaeQdNWKPAwlnkYIzB3Na6nUALnyFY-ki1euFGU.F3Zs36mJlugS4fB-6CtqbQ"
+        },
+    
+        "record": {
+            "CNAME": "asteral1.github.io/asteral"
```

```suggestion
            "CNAME": "asteral1.github.io"
```

CNAMEs cannot contain slashes (`/`).